guides/common/modules/snip_prerequisites-upgrading-project-server.adoc

```diff
@@ -12,6 +13,10 @@ Cloning is the recommended method because that prevents them being overwritten i
 To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
 In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
 If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.
+ifdef::satellite[]
+* Optional: Clone your {ProjectServer} to test the upgrade.
```

Comment:
I'm not sure if testing the upgrade really is the only purpose of cloning the server but it's what the corresponding chapter in Administering Project suggests.

Reply:
Can be both. I think we also support upgrades via cloning, but normally I look at @evgeni for those questions.

Reply:
To give a bit of context: I'm adding this because the [3.3 version of the document](https://docs.theforeman.org/3.3/Upgrading_and_Updating/index-satellite.html#upgrade_paths) states cloning the server as the first step. But I want to add a reason for doing it, especially since by making it optional we are asking users to decide whether they want to do it.

Reply:
The user can certainly do both:
* create a clone, use that for testing the upgrade, then upgrade the original box and throw away the clone
* create a clone, upgrade it, test it, promote it to "the real thing" and throw away the original

Reply:
Thanks! I added a few words to describe the "why". I'm pretty sure this is not the best place to add this information but my immediate goal is to add a step about cloning + explain when & why to clone so I think this is good enough for now. Tagging @apinnick in case she wants to look into this when planning the upgrade guide improvements.